scrapers/DDFNetwork.yml

```diff
@@ -0,0 +1,27 @@
+name: DDFNetwork
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - ddfnetwork.com/videos/
```

Comment:
ddfnetwork.com moved to pornworld.com
so
`      - pornworld.com/videos/`

Reply:
- It's **watch** not videos

Reply:
pornworld aleady exists as you said so we can just remove the ddfnetwork.com

Reply:
yes, i removed it.